api-reference/docstubs/api/user-list-calendarview.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "List calendarView"
+description: "Get the events from the calendarView navigation property."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List calendarView
```

Comment:
BTW, it would be interesting to see the calendar-list-calendarview.md topic too. 
I'd like to see if the generated topic would include the various syntax including a specific calendar other than the primary default calendar, or a specific calendar in a calendar group, as in the live [topic](https://docs.microsoft.com/en-us/graph/api/calendar-list-calendarview?view=graph-rest-beta&tabs=http#http-request).